srcs/incs/MimeTypes.hpp

```diff
@@ -0,0 +1,22 @@
+#ifndef SRCS_INCS_MIMETYPES_HPP_
+#define SRCS_INCS_MIMETYPES_HPP_
+#include <map>
+#include <string>
+
+class MimeTypes {
+	public:
+		static std::string			GetMimeType(const std::string &ext);
+
+	private:
+		// Map of file extensions and document types
```

Comment:
To be honest, this idea of using typedefs as a sort of comment, is already pervasive in our code... I've been looking at it, and although I don't like it, I don't have a very strong reason to refuse it.

Reply:
The important thing is that we all can understand what our code does, and a comment is as much of a form of documentation if not even better than a typedef (which I agree is a bit of a "hack" :smile:)